bot/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# startup script for wuestbot on gunicorn
+
+# Project Name
+NAME="wuestbot"
+
+# Host to listen on
+HOST=127.0.0.1
+
+# Port to listen on
+PORT=5000
+
+# Number of gunicorn Workers
+NUM_WORKERS=1
+
+# Databank directory
+DB_DIR=app/database
+
+# start venv
+source venv/bin/activate
+
+# create database
+# source $DB_DIR/db_create.py
+
+# startup server
+exec gunicorn $NAME:app \
```

Comment:
We should either create a check if the `API_TOKEN` env variable is set (and error out if it isn't), or allow passing the env var into the script, like with the other variables above.

Reply:
really unsure about this, since `API_TOKEN` is static and just lives in `.env`. can you elaborate on that?

Reply:
Well how is `start.sh` - and by an extend `gunicorn` - know about `.env` and the `API_TOKEN` ? We're not sourcing it in this script.

Option 1)

```shell
ENV_FILE=.env

#...
source venv/bin/activate

# create database
source $DB_DIR/db_create.py

# source .env 
source $ENV_FILE

# Check if API_TOKEN is set
if [ -z ${API_TOKEN+x}; then
  echo "API_TOKEN needs to be set"
  exit 1
fi

# startup server
exec gunicorn $NAME:app \
    -b $HOST:$PORT \
    -w $NUM_WORKERS \ 
```

Option 2)

```shell
#...
source venv/bin/activate

# create database
source $DB_DIR/db_create.py

# Check if API_TOKEN is set
if [ -z ${API_TOKEN+x}; then
  echo "API_TOKEN needs to be set"
  exit 1
fi

# startup server
exec gunicorn $NAME:app \
    -b $HOST:$PORT \
    -w $NUM_WORKERS \ 
```

```shell
API_TOKEN=abc ./start.sh

Reply:
currently we are calling it

Reply:
Yeah I would break that out from the bot code and just load them via `os.getenv()`. Then provide all environment variables up front. This decouples it from needing an env file.

Reply:
we'll work on that in a seperate pr